pipelines/longterm/pipeline.py

```diff
@@ -0,0 +1,65 @@
+"""Script for running the historic data pipeline."""
+
+from sys import stdout
+from logging import getLogger, StreamHandler, INFO
+
+from dotenv import load_dotenv
+
+from extract import get_data_from_rds
+from transform import get_summary_from_df
+from load import load_all
+
+
+def set_logger():
+    """Set logger."""
+    logger = getLogger()
+    logger.setLevel(INFO)
+    logger.addHandler(StreamHandler(stdout))
+
+
+def run():
+    """Run pipeline script."""
+    logger = getLogger()
+    logger.info("Attempting pipeline run...")
+
+    data = get_data_from_rds()
+    if data.empty:
+        raise ValueError("Recieved no data from RDS.")
+    logger.info("Successfully recieved data from RDS!")
+
+    summary = get_summary_from_df(data)
+    if summary.empty:
+        raise ValueError("Found no summary data from returned raw data!")
+    logger.info("Successfully summarised data from RDS!")
+
+    load_all(summary)
+    logger.info("Successfully loaded data into S3!")
+
+
+def lambda_handler(event, context):
```

Comment:
maybe want to set the defaults of these to None?
```suggestion
def lambda_handler(event=None, context=None):
```

Reply:
Just for local testing etc, but if you've tested and its all good I guess that's fine

Reply:
Do we want tests for the lambda handler? I was under the impression that we were not doing that.

Reply:
All good, I just meant if you were testing to see if it ran locally.